src/components/navbar/index.js

```diff
@@ -10,29 +11,41 @@ import './navbar.scss';
 
 const Navbar = () => {
     const [showLinks, setShowLinks] = useState(false);
-    const [showCart, setShowCart] = useState(false)
+    const [showCart, setShowCart] = useState(false);
 
+    const controlCart = () => {
+        if (!showLinks) {
+            setShowCart(!showCart);
+        }
+    }
+
+    const controlHamburger = () => {
+        if (!showCart) {
+            setShowLinks(!showLinks);
+        }
+    }
 
     return (
         <div className='nav__center'>
 
             <div className='nav__center__header'>
                 <img src={logo} className='nav__center__header__logo' alt='logo' />
                 <div className='nav__center__header__icons'>
-                    <button className='nav__center__header__icons__toggle nav__center__header__icons__toggle--cart' onClick={()=>setShowCart(!showCart)}>
+                    <button className='nav__center__header__icons__toggle nav__center__header__icons__toggle--cart' onClick={()=>controlCart()}>
                         <FaShoppingCart />
                     </button>
                     <span className='nav__center__header__icons__toggle'>2</span>
                     <BiDotsVertical className='nav__center__header__icons__toggle'/>
-                    <button className='nav__center__header__icons__toggle nav__center__header__icons__toggle--hamburger' onClick={()=>setShowLinks(!showLinks)}>
-                        <FaBars />
+                    <button className='nav__center__header__icons__toggle nav__center__header__icons__toggle--hamburger' onClick={()=>controlHamburger()}>
+                        {!showLinks && <FaBars />}
+                        {showLinks && <AiOutlineClose />}
```

Comment:
What a `showLinks`?

Reply:
I used showLinks just to show the links when hamburger is clicked. if it is clicked, i change it to true, by this logic, i change hamgurger to close(X) icon. I think I can rename it to showHamburger.